tests/requirements.txt

```diff
@@ -0,0 +1,106 @@
+#
```

Comment:
i don't really think we need this, or pip-compile can we remove it?

Reply:
Done, I decided to simulate a clean-install using pip only. If you now run `make up` it will basically try to auto-cleanup and force update dependencies.